src/ansi-c/ansi_c_declaration.h

```diff
@@ -59,20 +59,20 @@ class ansi_c_declaratort : public nullary_exprt
 
 inline ansi_c_declaratort &to_ansi_c_declarator(exprt &expr)
 {
-  assert(expr.id()==ID_declarator);
+  PRECONDITION(expr.id() == ID_declarator);
```

Comment:
Good change, but seems not related to the main goal of this commit/PR. Please put all the code cleanup done here in a separate PR.

Reply:
I removed these changes and I'll submit it on a separate PR.